esphome/components/sm2135/__init__.py

```diff
@@ -0,0 +1,31 @@
+import esphome.codegen as cg
+import esphome.config_validation as cv
+from esphome import pins
+from esphome.const import (
+    CONF_CLOCK_PIN,
+    CONF_DATA_PIN,
+    CONF_ID,
+)
+
+AUTO_LOAD = ["output"]
```

Comment:
Please add yourself as a Codeowner and run `script/build_codeowners.py`

```suggestion
AUTO_LOAD = ["output"]
CODEOWNERS = ["@BoukeHaarsma23"]

```